docs/release-notes.txt

```diff
@@ -5,9 +5,92 @@
 ###############
 
 **************
- Version 0.15
+ Version 0.16
 **************
 
+Version 0.16.0
+==============
+
+**Release Date:** June 14, 2021
+
+**New Features**
+
+-  Python SDK: Extend the :ref:`Checkpoint Export API
+   <exporting-checkpoints>` into a Python SDK capable of launching and
+   controlling experiments on the cluster directly from Python. See the
+   docs and examples in the :mod:`~determined.experimental.client`
+   module.
+
+-  Trials: Add new support for profiling model code. For all frameworks,
+   collecting system metrics, such as GPU utilization and memory, is
+   supported. For PyTorch, additional profiling for timing is available.
+   To quickly try out profiling, set ``profiling.enabled = true`` in the
+   experiment configuration. For a longer introduction to the features,
+   see the :ref:`guide on profiling <how-to-profiling>` or view details
+   on available configuration options in the :ref:`reference
+   <experiment-configuration_profiling>`.
+
+-  Experiments: Add new ``notes`` and ``name`` fields to experiments.
+
+-  REST API: Add new parameters to ``/api/v1/experiments`` to filter and
+   sort experiments by name.
+
+-  Master configuration: Support ``bind_mounts`` in
+   ``task_container_defaults`` in the master configuration. The
+   configured directories will be mounted for experiments, notebooks,
+   commands, shells, and TensorBoards.
+
+-  Images: Add an environment image containing TensorFlow 2.5 and CUDA
```

Comment:
non-blocking: should we link to the environments repo for this?

Reply:
Eh, it doesn't look like the precedent is to have have any links for new images.